exercises/bowling/src/main/java/BowlingGame.java

```diff
@@ -0,0 +1,11 @@
+public class BowlingGame {
```

Comment:
Since the spot you have chosen to put this is >21, you can remove this file and replace it with a `.keep` file. (See the [POLICIES.md](https://github.com/exercism/java/blob/master/POLICIES.md#starter-implementations) file)